presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PruneUnreferencedOutputs.java

```diff
@@ -357,8 +357,11 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Set<Variab
 
                 if (context.get().contains(variable)) {
                     Aggregation aggregation = entry.getValue();
-                    expectedInputs.addAll(extractUniqueVariables(aggregation, symbolAllocator.getTypes()));
-                    aggregation.getMask().ifPresent(expectedInputs::add);
+                    expectedInputs.addAll(
+                            extractUniqueVariables(aggregation, symbolAllocator.getTypes())
```

Comment:
nit: Just do `ImmutableList.copyOf(extractUniqueVariables(aggregation, symbolAllocator.getTypes())`. ditto for line 661.

Reply:
Good catch, we don't need this conversion after the rebase. The stream/collect was needed because we need to convert to Symbol.